parser.js

```diff
@@ -31,7 +31,7 @@ module.exports = function parser(config = {}) {
         revertPattern: /^(?:Revert|revert:)\s"?([\s\S]+?)"?\s*This reverts commit (\w*)\./i,
         revertCorrespondence: ['header', 'hash'],
         issuePrefixes: ['#'],
-        mergePattern: /^Merge pull request #(\d+) from (.*)$/,
-        mergeCorrespondence: ['id', 'source']
+        mergePattern: /^Merge (branch '(.*)' into (.*)|pull request #(\d+) from (.*))$/,
+        mergeCorrespondence: ['source', 'destination', 'id', 'source']
```

Comment:
I'm not 100% convinced by the regex, I came across a page with several message examples which require a more complex one: https://www.regextester.com/103784

Also, don't you have 5 capturing groups in the pattern but 4 entries in the correspondence? You could ignore groups with `(?:)` if it helps.

I don't know how critical this part is but we might as well cover most known message formats.